worlds/shorthike/Items.py

```diff
@@ -50,7 +50,7 @@ class ItemDict(TypedDict):
     {"name": "27 Coins", "id": base_id + 33, "count": 1, "classification": ItemClassification.filler},
     {"name": "32 Coins", "id": base_id + 34, "count": 1, "classification": ItemClassification.filler},
     {"name": "33 Coins", "id": base_id + 35, "count": 6, "classification": ItemClassification.filler},
-    {"name": "50 Coins", "id": base_id + 36, "count": 1, "classification": ItemClassification.filler},
+    {"name": "50 Coins", "id": base_id + 36, "count": 1, "classification": ItemClassification.progression},
```

Comment:
If I'm reading this PR right, no rules refer to the 50 Coins item, so maybe it should be `progression_skip_balancing` or `useful`?

Reply:
Switched that and the other 30+ coins to useful